### ISSUE TITLE
Edited Bolos Are Not Returned In API

### ISSUE THREAD

**Issue by Denominator765** [SnailyCAD/snaily-cadv4#1532]
### What version of Node.js are you using?

10.19.0

### What operating system are you using?

Ubuntu

### What version of SnailyCAD are you currently using?

1.47.1

### Describe the Bug

Editing the plate of a bolo after it has been created does not change the results of bolo API requests.

### Expected Behavior

Bolo API requests (/bolos?query={new_bolo}) using the new bolo plate number should return the bolo status.

### To Reproduce

Create a bolo with the wrong plate number.
Edit the bolo to contain the correct plate number.
Make an API request (/bolos?query={new_bolo}) with the correct plate number.

**Comment by Dev-CasperTheGhost** [comment]
I'm unable to reproduce this. Can you provide a video reproduction of this?

**Comment by Denominator765** [comment]
Turns out I can't edit the bolo at all, as in it doesn't change in the UI either (I found this when I had a mix up between a zero and an o, so was difficult to tell in the UI.) (Due to file size limitations, had to put the video in Google Drive. https://drive.google.com/file/d/15swKHWhV9DOhEBDdhX9wHgzXLHlXG22h/view?usp=sharing)

As for the nodejs version, I am not 100% certain that is the version that Snaily is using, as I am not very familiar with connecting to docker containers to check their internal software versions. If the docker build pulled in a later nodejs version, then that should be the one that it is using. I just pulled the version native to my OS.
